### ISSUE TITLE
Keyboard tab stops with visible focus

### ISSUE THREAD

**Issue by claudiu-cristea** [dequelabs/axe-core#4262]
### Product

axe-core

### Question

I have a requirement to check for that keyboard tab-stops have visible focus when active. I've used the `wcag247` tag to test.  However, a page violating this rule is not reported.

I've tested this page https://joinup.ec.europa.eu/collection/govtechconnect/news/register-govtech-connect-govtech4all-webinar which is supposed to report that the `<button class="wt-btn" type="button" id="eTransButton" aria-controls="wtEtransModal">Translate</button>` button, even is a tab-stop don't show a visible focus when is activated by keyboard.

Or I'm missing something?

**Comment by straker** [comment]
IGT is available with a pro account or trail from the [axe DevTools Extension](https://docs.deque.com/devtools-html/4.0.0/en/devtools-extension).

**Comment by WilcoFiers** [comment]
Determining visible focus isn't something axe-core would be able to do without false positives. To do it well, axe-core would need to focus elements in the page. Axe-core is built to do single-state testing. If it were to start focusing elements on the page, that focus event could trigger a stage change, which axe-core wouldn't be able to handle.

So yeah, axe Pro IGTs are the way to go for now.
